project/src/main/career/ui/progress-board-clock-label.gd

```diff
@@ -0,0 +1,31 @@
+extends Label
+## Draws the analog clock which appears above the progress board.
+##
+## The clock shows text like '12:30 pm', and can flash and emit particles.
+
+## The duration that the clock should flash white.
```

Comment:
Explicitly state 'in seconds'

Reply:
Changed, `The duration in seconds that the clock should flash white`

Also changed a similar comment in MusicPopupTween